Avances Grupo 1

Monsalve-Nicolas opened the pull request:
-movimiento
-apuntar
-salto

Monsalve-Nicolas committed:
PlayerMove

-movimiento
-apuntar
-salto

Monsalve-Nicolas committed:
BonusManager

Script enfocado en el Managment del Bonus, el script mantiene comentarios entorno a las funciones que se deben completar.

elMat3r committed:
Apolo

- Script de funcionamiento de enemigo.
+Preparar script para patrullaje.
+Trabajar en Tilemap.

elMat3r commented:
Se avanzo con el sistema de movimiento del _Player_ y el funcionamiento básico del _Enemigo_
Con el _Player_ se cambiaron el sistema de apuntado y direcciones que tiene que tener.

Monsalve-Nicolas committed:
Merge branch 'elwatona:main' into main

Monsalve-Nicolas committed:
Merge branch 'main' of https://github.com/Monsalve-Nicolas/USEK-Arcade

Monsalve-Nicolas committed:
Assembly Update

Referencia a TMPro mediante assembly

NickShogun24 committed:
Cambios en BonusManager

Creación de funciones para orientar la creación y objetivo del BonusManager.

NickShogun24 committed:
Actualizacion pequeña

Ultimos comentarios del push reciente.

Monsalve-Nicolas committed:
Update player Script

- move por switch
-funcion de disparo

Monsalve-Nicolas committed:
Monsalve-Parte

Player2
script
bug jump

elMat3r committed:
Sesión en casa: 12- 11- 2024

Tilemap y comienzo del background

NickShogun24 committed:
Modificaciones en Scrips y escena bonus1

Creacion de script Player1 y Player 2, eliminacion de funciones en BonusManager

NickShogun24 committed:
Carpeta BonusScripts

NickShogun24 committed:
Cambios BonusScript

Monsalve-Nicolas committed:
Merge branch 'elwatona:main' into main

Monsalve-Nicolas committed:
BossScene

Boss- Boss Phases and Take Damge
Player2 - Shield, jump

NickShogun24 committed:
Modificaciones y Codigos Bonus

Se creo una carpeta Bonus para mas organización, canvas mas completo, script incompleto de trigger, aun faltan detalles.

NickShogun24 committed:
Actualizaciones hechas en ayudantía

Creación carpeta de media con canción hecha por Nick, Implementado a la escena audiosource, aun hace falta que funcione el script del trigger.

NickShogun24 committed:
Cambios

NickShogun24 committed:
Problemas con el texto de puntaje 1 y 2

Modificaciones para poder revisar si el trigger se activa de manera correcta pero los textos no parecen reaccionar.

elMat3r committed:
Sesion en casa: 17 - 11 - 2024

Edicion del mapa e implemento de enemigos

NickShogun24 committed:
Diseño en BonusLevel

elMat3r committed:
Sesión en casa: 18 - 11 - 2024

Enemigo Común

NickShogun24 committed:
Merge branch 'main' into Bonuslevel

NickShogun24 committed:
Avances diseño mapa y arreglo de textos

Monsalve-Nicolas committed:
Boss

player 1 y 2 con condicion de derrota
tilemap

elMat3r committed:
Sesión en clases: 18 - 11 - 2024

Trigger de final
Mejoramiento del enemigo

elMat3r committed:
Merge branch 'main' of https://github.com/Monsalve-Nicolas/USEK-Arcade

NickShogun24 committed:
Cambios

Error al mostrar el puntaje al terminar el bonus

Monsalve-Nicolas committed:
Boss

script daño y cambio de fase

NickShogun24 committed:
Merge branch 'main' into Bonuslevel

NickShogun24 committed:
Cambios

NickShogun24 committed:
Merge branch 'main' into Bonuslevel

NickShogun24 committed:
Cambios

Monsalve-Nicolas committed:
cambios

boss

NickShogun24 committed:
Merge branch 'main' into Bonuslevel

Monsalve-Nicolas committed:
Boss

Movimiento
disparo

elMat3r committed:
Merge branch 'elwatona:main' into main

NickShogun24 committed:
BonusCodigoTerminado

Unicamente falta el diseño de los personajes y el generador

Monsalve-Nicolas committed:
Cambios

cambios de nombres de variantes y de funciones

Monsalve-Nicolas committed:
Merge branch 'main' of https://github.com/Monsalve-Nicolas/USEK-Arcade

NickShogun24 committed:
Merge branch 'main' into Bonuslevel

NickShogun24 committed:
Arreglo texto y boton

NickShogun24 committed:
UltimosCambios

Monsalve-Nicolas committed:
Merge branch 'elwatona:main' into main

Monsalve-Nicolas committed:
boss

poner las variantes

NickShogun24 committed:
Merge branch 'main' into Bonuslevel

NickShogun24 committed:
Escena de pruebas de sprites y animator

Se agrego carpetas de assets en unity Pixel army 2D y 100 fantasy characters, escena para pruebas de assets

Monsalve-Nicolas committed:
Boss

daño al boss y al jugador

NickShogun24 committed:
Pruebas

Assets

NickShogun24 committed:
Scenas

elMat3r committed:
Sesion en clases: 25-11-2024

Arreglos nivel 1

elMat3r committed:
Merge branch 'main' of https://github.com/Monsalve-Nicolas/USEK-Arcade

Monsalve-Nicolas committed:
Revert "Scenas"

This reverts commit e0531c8160d259137fe40b676b3806bafa4b9d11.

Monsalve-Nicolas committed:
Merge branch 'main' of https://github.com/Monsalve-Nicolas/USEK-Arcade

NickShogun24 committed:
Update Finish

NickShogun24 committed:
Merge branch 'main' into Bonuslevel

NickShogun24 committed:
Carpeta EntitySpritesMain

Monsalve-Nicolas committed:
Intro

Texto de la intro y la finishline

Monsalve-Nicolas committed:
Merge branch 'main' of https://github.com/Monsalve-Nicolas/USEK-Arcade

elMat3r committed:
Sprite Player1

Testeos en escena pruebas

elMat3r committed:
Merge branch 'main' of https://github.com/Monsalve-Nicolas/USEK-Arcade

NickShogun24 committed:
Player2 Sprite

NickShogun24 committed:
Merge branch 'main' of https://github.com/Monsalve-Nicolas/USEK-Arcade

elMat3r committed:
Sesion en clases: 27-11-2024

Trabajos con los sprites

elMat3r committed:
Merge branch 'main' of https://github.com/Monsalve-Nicolas/USEK-Arcade

Monsalve-Nicolas committed:
Tilemap

tilemap

Monsalve-Nicolas committed:
Merge branch 'main' of https://github.com/Monsalve-Nicolas/USEK-Arcade

NickShogun24 committed:
Arreglo Codigo Bonus

Testeo de joysticks y arreglo en el bonus

NickShogun24 committed:
Update GME1_level1.unity

NickShogun24 committed:
Update GME1

Monsalve-Nicolas committed:
Level2

enemigos

NickShogun24 committed:
SpritesAplicados

Sprites puestos en Level1

NickShogun24 committed:
Cambios Bonus1

NickShogun24 committed:
Modificaciones Niveles Boss, Level1 y Bonus

Script que sigue al player 1

Monsalve-Nicolas committed:
erro

problemas con github

Monsalve-Nicolas committed:
erro

problemas con github

Monsalve-Nicolas committed:
nuevo boss

boss
